lib/reek/report/report.rb

```diff
@@ -129,9 +130,9 @@ def show
     class HTMLReport < Base
       require 'erb'
 
-      def show(target_path = 'reek.html')
-        template_path = File.expand_path('../html_report.html.erb', __FILE__)
-        File.write target_path, ERB.new(File.read(template_path)).result(binding)
+      def show(target_path = Pathname('reek.html'))
+        template_path = Pathname("#{__dir__}/html_report.html.erb")
```

Comment:
I wish Pathname did a little bit more its best to be like File and allowed `Pathname.join('foo', 'bar')`. None of these look very appealing, but let's go with 2. for now since that's what it is now.